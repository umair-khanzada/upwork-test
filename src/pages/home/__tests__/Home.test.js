import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../Home';
describe('MyComponent', () => {
  it('should render HomePage', () => {
    const component = shallow(<HomePage />);

    expect(component).toMatchSnapshot();
  });
});
