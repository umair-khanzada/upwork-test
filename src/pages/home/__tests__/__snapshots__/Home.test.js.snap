// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render HomePage 1`] = `
<styled__HomePage
  select="true"
>
  <Adapter>
    <Heading
      className="subHeading"
      level={2}
      titleText="Hi John we missed you."
    />
    <styled__HomeDesc>
      Click the plus button to start your advice
    </styled__HomeDesc>
    <styled__ButtonModalFixed
      shape="circle"
      size="large"
      type="primary"
    >
      <Icon
        type="plus"
      />
    </styled__ButtonModalFixed>
  </Adapter>
</styled__HomePage>
`;
